### PR TITLE
Setup Sync 2.0 on UgandaEMR

### PR DESCRIPTION
**Pull request by echiteri**
FHIR module version to 1.18.0
webservices.restModuleVersion to 2.24.0
Added:
Atomfeed module version 1.0.12
Sync2 module version 1.6.0

### CODE REVIEW

File: omod/src/main/resources/config.xml [hunk 1]
```diff
@@ -84,6 +84,9 @@
         <require_module>
             org.openmrs.module.uiframework
         </require_module>
+        <require_module version="${sync2Version}">
```
Comment (ssmusoke) [inline omod/src/main/resources/config.xml]: @echiteri use aware of so that when the module is stopped things do not break

File: api/pom.xml [hunk 2]
```diff
@@ -141,6 +141,11 @@
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>legacyui-omod</artifactId>
 		</dependency>
+		<dependency>
```
Comment (ssmusoke) [inline api/pom.xml]: @echiteri Add to alphabetical location

File: pom.xml [hunk 3]
```diff
@@ -79,11 +79,13 @@
 		<jacksonVersion>1.8.3</jacksonVersion>
 		<junitParamsVersion>1.0.4</junitParamsVersion>
 		<legacyuiVersion>1.4.0</legacyuiVersion>
+		<sync2Version>1.6.0</sync2Version>
+		<atomfeedVersion>1.6.0</atomfeedVersion>
```
Comment (ssmusoke) [inline pom.xml]: @echiteri Atom feed does not have a 1.6.0 version

File: omod/src/main/resources/config.xml (hunk 1 above)
Comment (echiteri) [reply omod/src/main/resources/config.xml]: @ssmusoke noted

File: pom.xml (hunk 3 above)
Comment (echiteri) [reply pom.xml]: @ssmusoke I updated this

File: omod/pom.xml [hunk 4]
```diff
@@ -34,6 +34,18 @@
...
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>atomfeed-omod</artifactId>
+			<version>${atomfeedVersion}</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>sync2-omod</artifactId>
+			<version>${sync2Version}</version>
```
Comment (ssmusoke) [inline omod/pom.xml]: @echiteri Remove the version since this read from the parent pom.xml

File: omod/pom.xml [hunk 5]
```diff
@@ -34,6 +34,18 @@
 			<type>test-jar</type>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>atomfeed-omod</artifactId>
+			<version>${atomfeedVersion}</version>
```
Comment (ssmusoke) [inline omod/pom.xml]: @echiteri Remove the version since this read from the parent pom.xml

File: api/pom.xml [hunk 6]
```diff
@@ -66,6 +66,12 @@
 			<artifactId>appframework-api</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>atomfeed-omod</artifactId>
+			<version>${atomfeedVersion}</version>
```
Comment (ssmusoke) [inline api/pom.xml]: @echiteri Remove the version since this read from the parent pom.xml

File: api/pom.xml [hunk 7]
```diff
@@ -192,6 +198,12 @@
 			<artifactId>serialization.xstream-api-2.0</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>sync2-omod</artifactId>
+			<version>${sync2Version}</version>
```
Comment (ssmusoke) [inline api/pom.xml]: @echiteri Remove the version since this read from the parent pom.xml

File: omod/pom.xml [hunk 8]
```diff
@@ -85,6 +84,12 @@
 			<artifactId>appui-omod</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>atomfeed-omod</artifactId>
+			<version>${atomfeedVersion}</version>
```
Comment (ssmusoke) [inline omod/pom.xml]: @echiteri Remove the version since this read from the parent pom.xml

File: omod/pom.xml [hunk 9]
```diff
@@ -216,6 +221,12 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>sync2-omod</artifactId>
+			<version>${sync2Version}</version>
```
Comment (ssmusoke) [inline omod/pom.xml]: @echiteri Remove the version since this read from the parent pom.xml

**Comment by ssmusoke** [comment]
@echiteri Change the title of the PR as discussed

File: api/src/main/resources/liquibase.xml [hunk 10]
```diff
@@ -217,4 +217,17 @@
...
 	</changeSet>
+
+	<changeSet id="ugandaemr-30-04-2019-1524" author="echiteri">
+		<comment>Setting  modules to Atomfeed, sync 2.0 to be started manually</comment>
+		<sql>
+			UPDATE global_property SET property_value = 'false' WHERE property IN ('atomfeed.started', 'sync2.started');
+		</sql>
+	</changeSet>
+	<changeSet id="ugandaemr-30-04-2019-1524" author="echiteri">
+		<comment>Persist global propertis for sync2 </comment>
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml]: @echiteri This comment should be talking about why this merge behavior is selected over the default

**Pull request closed without merging**